sparkle/src/components/Breadcrumbs.tsx

```diff
@@ -0,0 +1,52 @@
+import Link from "next/link";
```

Comment:
You can't use next/link in sparkle (no next dependency here). Maybe we should pass the component to render for link as a prop ?